example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -76,12 +78,9 @@ bool offb_ctrl_ned(Device &device)
 
```

Comment:
@julianoes In line above, why sleep for one second after starting offboard? API should not require artificial timing breaks.

Reply:
Removed.